src/Directory.Build.props

```diff
@@ -61,12 +61,18 @@
     <NuGetCmd>$(NuGetRoot)nuget.exe</NuGetCmd>
     <!-- Respect environment variable for the .NET install directory if set; otherwise, use the current default location -->
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);NU1901;NU1902;NU1903;NU1904;NU1905</WarningsNotAsErrors>
```

Comment:
@cheenamalhotra doesn't this cause the CI build to pass even if there are dependency vulnerability warnings?

Reply:
I believe it enables us and open source contributors to capture warnings and address vulnerabilities when feasible without breaking builds. For any high/critical vulnerability, it will be captured by CG anyways so we will be addressing them on priority.